blocks/header/header.css

```diff
@@ -144,7 +154,8 @@ header nav .nav-sections ul > li > ul > li {
   font-weight: 500;
 }
 
-@media (width >= 900px) {
+/* Desktop view (--udexGridMMinWidth) */
+@media (width >= 980px) {
```

Comment:
I had made all these changes for header and a lot more. There might be a lot of conflicts. Would it be better to remove header.css from this PR?